prototype/Makefile

```diff
@@ -40,3 +44,7 @@ clean_all: clean_venv
 .PHONY: run
 run:
 	paster
+
+.PHONY: test
+test:	deps
+	bin/nosetests -x -s jenkinsmerger
```

Comment:
If it isn't used I'd propose killing it off.  When looking for the right place to make the change 'git grep' led me to this part of the code base.  If we do keep it, perhaps we should update the README to have something useful.